packages/swirl-components/src/components/swirl-shell-navigation-item/swirl-shell-navigation-item.css

```diff
@@ -107,6 +107,23 @@
       aspect-ratio: 1;
     }
 
+    &.shell-navigation-item--gradient {
+      & .shell-navigation-item__icon {
+        &::after {
+          content: "";
+          position: absolute;
+          width: 100%;
+          height: -webkit-fill-available;
```

Comment:
Issue: Doesn't seem to work in Firefox (non-webkit browsers).

Reply:
yeah it doesn't, forgot to replace it